cotel/src/main/scala/day4/NoAnagramPassphrasses.scala

```diff
@@ -0,0 +1,29 @@
+package day4
+
+import scala.io.Source
+
+object NoAnagramPassphrasses {
+
+  def isValidPassphrase(passphrase: String, checked: List[String] = List(), result: Boolean = true): Boolean = {
+    def recursion(passphrase: List[String], checked: List[String] = List(), result: Boolean = true): Boolean =
+      (passphrase, checked, result) match {
+        case (List(), _, acc) => acc
+        case (x :: xs, ys, acc) if !ys.contains(x) => recursion(xs, x :: ys, acc)
```

Comment:
For convention, always put the recursion in the last position 👍 

I think you can use tailrec 👍

Reply:
Yes, you are right 💯